setup.py

```diff
@@ -53,7 +53,6 @@
     'snowballstemmer>=1.1',
     'babel>=1.3,!=2.0',
     'alabaster>=0.7,<0.8',
-    'sphinx_rtd_theme>=0.1,<2.0',
```

Comment:
I've updated the PR as suggested. Now it gives the following error if `sphinx_rtd_theme` is not available:
```
Theme error:
sphinx_rtd_theme has been unbundled since version 1.4.0. Please install it manually.
```
Note that I've put 1.4.0 as the version number, but this can be easily changed.